source/server-discovery-and-monitoring/server-monitoring.rst

```diff
@@ -1143,29 +1139,22 @@ awaitable hello or legacy hello heartbeat in the new protocol.
 Changelog
 ---------
```

Comment:
Formatting is inconsistent with other files
```suggestion
=========
```

Reply:
This is consistent with the header syntax for "Abstract".